gravitee-apim-portal-webui/src/app/pages/dashboard/dashboard.component.ts

```diff
@@ -72,13 +69,34 @@ export class DashboardComponent implements OnInit {
     });
     if (this.hasApplicationPermission()) {
       this.applicationService
-        .getApplications({ size: 3, order: '-nbSubscriptions' })
+        .getApplications({ size: 3, order: 'nbSubscriptions' })
```

Comment:
I'm not sure to understand the point of reversing order here

Reply:
I think, the desired order and the use of prefix `-` was wrong. 
Here, we want the 3 applications with the highest nbSubscripttions.